WooCommerce/WooCommerceTests/ViewRelated/Products/Actions Factory/ProductFormActionsFactory+ProductCreationTests.swift

```diff
@@ -7,7 +7,7 @@ import Fakes
 final class ProductFormActionsFactory_ProductCreationTests: XCTestCase {
     func test_product_type_is_editable_for_core_product_types() {
         // Product types supported in core.
-        let coreProductTypes: [ProductType] = [.simple, .variable, .affiliate, .grouped]
+        let coreProductTypes: [ProductType] = [.simple, .variable, .affiliate, .grouped, .subscription, .variableSubscription]
```

Comment:
Here technically subscription/variable subscription is not a core product type, instead it's a type supported in Woo Extensions. However the test is to confirm that it is editable in the app, so indeed they might belong here.

wdyt about splitting it into its own variable? 

```
let nonCoreButEditableProductTypes: [ProductType] = [.subscription, .variableSubscription]
```

Or maybe include it in a separate unit test? e.g:

```
func test_product_type_is_editable_for_non_core_product_types() {
...
}
```

Reply:
Nice observation, Hafiz! I moved the product types of this core unit test. https://github.com/woocommerce/woocommerce-ios/pull/11248/commits/ed0229bf377e1a30a76a08b0fe96ada480afa9f3